core/src/context_menu.rs

```diff
@@ -34,9 +34,13 @@ impl<'gc> ContextMenuState<'gc> {
         &self.callbacks[index]
     }
     pub fn build_builtin_items(&mut self, item_flags: BuiltInItemFlags, stage: Stage<'gc>) {
-        let root_mc = stage.root_clip().as_movie_clip();
+        let root_mc = if let Some(root_clip) = stage.root_clip().and_then(|c| c.as_movie_clip()) {
+            root_clip
+        } else {
+            return;
+        };
```

Comment:
nit: This can just be
```suggestion
        let Some(root_mc) = stage.root_clip().and_then(|c| c.as_movie_clip()) else {
            return;
        };
```

Reply:
I had no idea that syntax worked!